models/stg_recharge__subscription_history.sql

```diff
@@ -32,14 +32,14 @@ final as (
         price,
         quantity,
         status,
-        property_charge_interval_unit_type as charge_interval_unit_type,
+        property_charge_interval_unit_type as charge_interval_unit_type, 
```

Comment:
Same comment about these all being custom fields. We should let the variables handle these.